WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginHeaderViewHolder.java

```diff
@@ -53,6 +55,7 @@ public void updateLoggedInAsHeading(Context context, boolean isAfterLogin, Strin
                 mDisplayNameTextView.setText(username);
             }
         } else {
+            imageManager.cancelRequestAndClearImageView(mAvatarImageView);
```

Comment:
Is there a reason we're clearing this? Just trying to think of the use case here on why we'd do this. 

Looks like the only place we're calling this with `isAfterLogin` false is `ShareIntentReceiverFragment.refreshAccountDetails()`